src/analyzer.ts

```diff
@@ -198,15 +214,17 @@ export class BuildAnalyzer {
    * Return _dependenciesOutputStream, which will contain fully loaded file
    * objects for each dependency after analysis.
    */
-  get dependencies(): NodeJS.ReadableStream {
+  dependencies(): NodeJS.ReadableStream {
```

Comment:
Oh, I see this is on analyzer anyway, which isn't often the public interface. I take back the churn comment.